libs/components/ag-grid/src/lib/styles/ag-grid-styles.scss

```diff
@@ -16,11 +17,138 @@ $sky-cell-wrap-text-line-height-modern: $sky-theme-modern-font-paragraph-line-he
 
 @mixin skyux-cell-wrap-text($padding, $line-height) {
   .ag-cell.ag-cell-wrap-text span.ag-cell-value {
-    display: block;
-    min-height: ag-param(row-height) - 2px;
-    line-height: $line-height;
+    line-height: #{$line-height};
     padding-top: $padding;
     padding-bottom: $padding;
+    min-height: var(--ag-row-height) - 2px;
+  }
+}
+
```

Comment:
This switches over to the new API for Sass themes ([docs](https://www.ag-grid.com/angular-data-grid/global-style-upgrading-to-v28-sass/)).